README.md

```diff
@@ -89,13 +89,13 @@ Some of the options available:
 
 | Option       | Description |
 |--------------|-------------|
-| :model       | Model to set errors
-| :attribute   | Model attribute to receive errors (default :base)
-| :message     | Custom error message
+| :model       | Model to set errors.
+| :attribute   | Model attribute to receive errors. (default :base)
+| :message     | Custom error message.
 | :private_key | Override private API key.
 | :timeout     | The number of seconds to wait for reCAPTCHA servers before give up. (default `3`)
-| :response    | Custom response parameter (default: params['g-recaptcha-response'])
-
+| :response    | Custom response parameter. (default: params['g-recaptcha-response'])
+| :domain      | Domain name to check equality when you need custom [domain name validation](https://developers.google.com/recaptcha/docs/domain_validation). You can also pass a callable that receives the [hostname field](https://developers.google.com/recaptcha/docs/verify#api-response) and returns if domain name is valid or not. (default: `nil`)
```

Comment:
How about:

Expected hostname or a callable that validates the hostname, see [domain validation](https://developers.google.com/recaptcha/docs/domain_validation) and [hostname](https://developers.google.com/recaptcha/docs/verify#api-response) docs.  (default: `nil`)